Configurations/descrip.mms.tmpl

```diff
@@ -179,7 +179,8 @@ MODULESDIR_C={- platform->osslprefix() -}MODULES{- $sover_dirname.$target{pointe
 
 CC={- $config{CC} -}
 CPP={- $config{CPP} -}
-DEFINES={- our $defines1 = join('', map { ",$_" } @{$config{CPPDEFINES}}) -}
+DEFINES={- our $defines1 = join('', map { ",$_" } ('OPENSSL_BUILDING_OPENSSL',
+                                                   @{$config{CPPDEFINES}})) -}
```

Comment:
I dislike this hard-coded stuff. The better thing would be to define it once, here:

https://github.com/openssl/openssl/blob/master/Configurations/00-base-templates.conf#L13

``` perl
        lib_defines => [ 'OPENSSL_BUILDING_OPENSSL' ], 
```

Reply:
Fixed in #10666 as commit dc83fb44e5f6916c64f25585461e77edc7c0a914 and then rebased this.